src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -47,44 +52,45 @@
                                                         .setCalendarType("iso8601")
                                                         .setDate(2020, 7, 18)
                                                         .build();
-    private final TutorSession bernardoSession = new TutorSession("btrevisan@google.com", "btrevisan@google.com", null, null, TimeRange.fromStartToEnd(540, 600, MAY182020), 0); 
-    private final TutorSession samSession = new TutorSession("sfalberg@google.com", "sfalberg@google.com", null, null, TimeRange.fromStartToEnd(540, 600, AUGUST182020), 1);                                         
+    //9 and 14 are the ids local datastore gives to these entities
+    private final TutorSession bernardoSession = new TutorSession("1", "1", null, null, TimeRange.fromStartToEnd(540, 600, MAY182020), 9); 
+    private final TutorSession samSession = new TutorSession("2", "2", null, null, TimeRange.fromStartToEnd(540, 600, AUGUST182020), 14);
```

Comment:
How does it decide on 9 and 14?

Reply:
I'm 90% sure the id is assigned by what order the entity is put into datastore. For these TutorSession entities, the first one is the 9th entity that gets put into datastore and the second one is the 14th. This is only relevant to the local datastore and SampleData. So if we decide to change SampleData and add more entities before these two session, we would need to update the ids.

Reply:
Oh okay, that makes sense

Reply:
The would probably account for this right? I mean if we add more entities and the id number changes

Reply:
Yeah, I'm not really sure of the best way to handle this besides manually changing the ids every time we add something to SampleData. It shouldn't affect the ids unless we add new entities **before** these tutoring sessions.